IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -60,14 +60,12 @@
                         <td>@brr.Longitude</td>
                         <td>@brr.Latitude</td>
                         <td>@brr.Phone</td>
-                        <td>@brr.WebsiteUrl</td>
-                        <td>@brr.UpdatedAt</td>
-                        <td>@brr.CreatedAt</td>
-
+                        <td><a href="@brr.WebsiteUrl">@brr.WebsiteUrl</a></td>
+                        <td>@brr.UpdatedAt.Date.Date.ToShortDateString()</td>

```

Comment:
"Created Date" and Updated Date" attributes display Date, Time as well as UTC details. I changed the code in such a way that it only displays the Date part which is what a user might just be interested in.